### PR TITLE
Add config option to ignore specific refresh groups

### PR DESCRIPTION
**Pull request by pierre**
This fixes https://github.com/killbill/killbill-analytics-plugin/issues/87.

/cc @beccagaspard

### CODE REVIEW

File: .idea/codeStyles/Project.xml [hunk 1]
```diff
@@ -0,0 +1,57 @@
+<component name="ProjectCodeStyleConfiguration">
```
Comment (sbrossie) [inline .idea/codeStyles/Project.xml]: I suppose this is not really part of the PR, always a bit confused with InteeliJ files..

**Comment by pierre** [comment]
> 1. Should we offer this functionality in the analytics plugin -- and potentially all other plugins -- or at the KB core level, where we could specify which events to send (all by defaults) or omit?

Note that the filtering isn't really done on a per event basis here, but rather based on the set of tables being refreshed (see below). Also, the motivation behind this patch is really to easily implement a priority scheme in the Analytics plugin (some tables matter more than others).

Regarding offering a way to filter out events in the core on a per plugin basis, we could, but I'm not sure if the performance impact would be big enough to justify the work. Once the event has been transformed from internal to external bus (and persisted), the hard work is done. Skipping it in the OSGI core (before send) or right in the plugin event handler (upon receive) probably doesn't make a noticeable difference.

It would be nice to ignore some events right before they are sent to the external bus if we know they are not going to be used by any plugin and the HTTP listener(s) (if any), but it's hard to get it right (the user would have to really understand how each event may be used by all plugins deployed).

> 2. The [mapping for group -> event](https://github.com/killbill/killbill-analytics-plugin/blob/bcf7d35d535d3c08566bc99cd83b550c936330a1/src/main/java/org/killbill/billing/plugin/analytics/AnalyticsJobHierarchy.java#L24) seems a bit weird. Mostly i would expect expect a group `ALL` to be all events, and not just `ACCOUNT` and `TAG`

`Group` is an Analytics plugin concept, mapping the _group_ of business tables being refreshed. `ALL` means refresh all of them, and only `ACCOUNT` and `TAG` related events require this (because of denormalization).

File: .idea/codeStyles/Project.xml (hunk 1 above)
Comment (pierre) [reply .idea/codeStyles/Project.xml]: > I suppose this is not really part of the PR

It was checked-in on purpose, in a different commit! 😉 

> always a bit confused with InteeliJ files..

Nowadays, `.gitignore` files have been tweaked so that only files that matter are checked-in (e.g. code style). If there are changes, it's very likely JetBrains decided to change some stuff (new naming for specific properties, how files are organized, etc.). So in most cases, it's safe to commit the updates but going through it is always good: in case of that recent update, they pushed for Java 8 refactoring, so I made sure `Anonymous2MethodRef`, `Convert2Lambda`, etc. is set to `false`.
